packages/css-jss/src/index.d.ts

```diff
@@ -0,0 +1,7 @@
+import {Style} from 'jss'
+
+export {Style}
+
+export type StyleArg = Style | Array<Style>
+
+export type Css = (...args: StyleArg[]) => string
```

Comment:
The definitions for `createCss` and the default export are missing here. These definitions will not work by it's own

Reply:
Done